src/kata4.humanCatDogYears.js

```diff
@@ -1,3 +1,14 @@
-const humanCatDogYears = (number) => {};
+const humanCatDogYears = (humanYears) => {
+  if (humanYears === 1) {
```

Comment:
This is pretty good, though what if the human years value passed in was 0? This should be taken into account and a test added to validate the output is what it should be (`[0, 0, 0]`)